app/models/manageiq/providers/kubevirt/inventory/parser.rb

```diff
@@ -164,14 +164,24 @@ def process_domain(memory, cores, uid, name)
 
     # Create the inventory object for the hardware:
     hw_object = hw_collection.find_or_build(vm_object)
-    hw_object.memory_mb = ManageIQ::Providers::Kubevirt::MemoryCalculator.convert(memory, 'Mi')
+    hw_object.memory_mb = parse_quantity(memory) / 1.megabytes.to_f
```

Comment:
Do we have tests exercising this for different extensions?  Only asking because I tried some stuff locally and it fails with expected values as per the original tests.

```
irb(main):008:0> "1 Ki".iec_60027_2_to_i
=> 1024
irb(main):009:0> "1 KB".iec_60027_2_to_i
Traceback (most recent call last):
        4: from bin/console:14:in `<main>'
        3: from (irb):9
        2: from /Users/jfrey/dev/more_core_extensions/lib/more_core_extensions/core_ext/string/iec60027_2.rb:11:in `iec_60027_2_to_i'
        1: from /Users/jfrey/dev/more_core_extensions/lib/more_core_extensions/core_ext/string/iec60027_2.rb:11:in `Integer'
ArgumentError (invalid value for Integer(): "1 KB")
irb(main):010:0> "1 KB".decimal_si_to_f
Traceback (most recent call last):
        5: from bin/console:14:in `<main>'
        4: from (irb):9
        3: from (irb):10:in `rescue in irb_binding'
        2: from /Users/jfrey/dev/more_core_extensions/lib/more_core_extensions/core_ext/string/decimal_suffix.rb:10:in `decimal_si_to_f'
        1: from /Users/jfrey/dev/more_core_extensions/lib/more_core_extensions/core_ext/string/decimal_suffix.rb:10:in `Float'
ArgumentError (invalid value for Float(): "1 KB")
```

I expected the last one to "work", but maybe I'm passing the wrong values?

Reply:
I don't think that "1 KB" is the format that kubernetes uses, here is how they parse capacity fields https://github.com/ManageIQ/manageiq-providers-kubernetes/blob/7ba567b0d2af94c71a85ccb54c4dde088f128e99/app/models/manageiq/providers/kubernetes/inventory/parser/container_manager.rb#L817-L836

Reply:
Interesting.  Wonder why the specs here had strings like `"1KB"`, then.

Reply:
yeah I have no idea, if you want I can put back my original fix which just updates the regex